app/models/terms_and_conditions.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class TermsAndConditions < ActiveRecord::Base
+  has_many :terms_and_conditions_acceptances
+
+  scope :latest, -> { find_by(latest: true) }
+
+  validates :name, presence: true
```

Comment:
But it's not really an STI problem, since there's really no point that I'm aware of in having a separate model for each one.